core-addon/content-script.js

```diff
@@ -22,6 +22,25 @@ function sendAddonAliveEvent() {
   sendToPage({type: "web-check-response", data: {}})
 }
 
+/**
+ * Convenience function to send a message to the Core Add-on.
+ *
+ * @param {String} type
+ *        The message type/name.
+ * @param {Object} data
+ *        An object containing the data for the message.
+ * @returns {Promise} resolved when the message is sent.
+ */
+async function sendToCore(type, data) {
+  // TODO: can we automatically get the addon id for the right platform
+  // at build time?
```

Comment:
Not here and now, but yes we should do this :) Also any URLs that are duplicated elsewhere too.

Reply:
I filed https://github.com/mozilla-rally/rally-core-addon/issues/409 !